api/endpoints/singlepoint_route.py

```diff
@@ -0,0 +1,63 @@
+"""Contains routes for performing singlepoint calculations."""
+
+from __future__ import annotations
+
+import logging
+from pathlib import Path
+from typing import Annotated, Any
+
+from fastapi import APIRouter, HTTPException, Query
+
+from api.utils.singlepoint_helper import singlepoint
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/singlepoint", tags=["calculations"])
+
+
+@router.get("/")
+def get_singlepoint(
+    struct: Annotated[str, Query()],
+    arch: Annotated[str | None, Query()] = "mace_mp",
+    properties: Annotated[list[str] | None, Query()] = None,
```

Comment:
This could similarly be a list of the properties `Literal` - see https://github.com/stfc/janus-core/blob/main/janus_core/calculations/single_point.py#L99